src/main/java/com/google/sps/servlets/ProfileImage.java

```diff
@@ -63,9 +61,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
       String userId = userService.getCurrentUser().getUserId();
-      Key userKey = KeyFactory.createKey("User", userId);
-      Entity userEntity = datastore.get(userKey);
-      fileName = (String) userEntity.getProperty("avatarName");
+      fileName = userId + ".png";
```

Comment:
If you store whether the user has previously uploaded a profile image, you can decide where whether to make filename `userID + ".png"` or `"default.png"`.